enclave/chain_relay/src/justification.rs

```diff
@@ -141,16 +110,15 @@ impl<Block: BlockT> GrandpaJustification<Block> {
 		let mut buf = Vec::new();
 		let mut visited_hashes = HashSet::new();
 		for signed in self.commit.precommits.iter() {
-			if communication::check_message_sig_with_buffer::<Block>(
+			if !sp_finality_grandpa::check_message_signature_with_buffer(
```

Comment:
Is the negation here correct?

Reply:
Yes, the signature is only bad if `false` is returned. https://github.com/paritytech/substrate/blob/master/client/finality-grandpa/src/justification.rs#L155

Reply:
Ok, thanks for confirming. Just wanted to make sure because it was different before 😄